### PR TITLE
Remove akka-camel

### PR DESCRIPTION
**Pull request by raboof**
Not for merge until we've branched `release-2.5` off.

Refs #26191

This module has been deprecated since 2.5 and will now be removed in 2.6.

If there is interest it can be moved to a separate community-maintained repo.

### CODE REVIEW

**Review by chbatey — COMMENTED** [review]
Note to migration guide then LGTM

File: akka-docs/src/main/paradox/project/migration-guide-2.5.x-2.6.x.md [hunk 1]
```diff
@@ -1,5 +1,14 @@
 # Migration Guide 2.5.x to 2.6.x
 
+## akka-camel removed
+
+After being deprecated in 2.5, the akka-camel module has been removed in 2.6.
```
Comment (patriknw) [inline akka-docs/src/main/paradox/project/migration-guide-2.5.x-2.6.x.md]: in 2.5.0

File: build.sbt [hunk 2]
```diff
@@ -43,7 +43,6 @@ lazy val aggregatedProjects: Seq[ProjectReference] = List[ProjectReference](
   agent,
   benchJmh,
   benchJmhTyped,
-  camel,
```
Comment (patriknw) [inline build.sbt]: also remove from `allModules` in ActorSystem.scala

**Review by chbatey — APPROVED** [review]
LGTM

**Review by patriknw — APPROVED** [review]
LGTM

**Pull request merged**